### DISCUSSION TITLE
Now Tv compatibility

### DISCUSSION THREAD

**Discussion by TransGirlSupremacy** [f1multiviewer/issue-tracker#495] [Feature requests]
hey yall, so Now Tv in the uk has the ability to view each drivers cams and like a multi screen timing thing now, i was wondering if yall were able or are thinking about adding Now Tv compatibility to the multiviewer app? If not thats cool

**Comment by edmerrett** [comment]
+1 on this

If you have a valid NowTV or Sky subscription you can get all the same views as F1 TV Pro.

**Comment by odiinh** [comment]
Absolutely another +1,

NOW TV (or Sky/Sky Go) is the only way brits can get access to the same feeds that F1 TV Pro do. I'd love to use multiviewer and this is the only thing holding me back. 

The only question is integrating live timing as NOW TV customers don't get access to a livetiming api. However would access to the OpenF1 API be viable?

**Comment by LapsTimeOFF** [comment]
Integrating NowTV is not in our plans at the moment, supporting 3rd-party services is quite a complex task and won't be integrated for now.
